packages/storybook/src/stories/select.stories.svelte

```diff
@@ -57,19 +54,21 @@ import {
   </Label>
 </Story>
 
-<Story name="Multi">
-  <Label cx="h-[200px]">
-    Select multiple objects
-    <Multiselect
+<Story name="Searchable with multiple">
+  <Label cx="h-[200px] max-w-[200px]">
```

Comment:
Not for this PR but I am seeing us use `200px` a lot as an arbitrary value, starting to feel like a `h-50`/`w-50` class might be worth it at some point.

Reply:
Yeah it's mostly just coming up in the stories. I'm not recalling any other places.

Reply:
I never understood this, but why did we settle on a max width of 200px?

Reply:
Design typically wants these inputs to have a max width of something like 200px instead of full width. The exact width is not something we want to prescribe in the component, however. We want the stories to demonstrate that to indicate proper usage.

Reply:
Ahh makes sense, thanks for explaining